PDR-0011-Project-Committee/roledescription.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: Project Committee Role
+Category: Guidelines
+Status: Pending
+Authors:
+    - Christina Gianelloni <christina@blinklabs.io> 
+Created: 2025-11-03
+---
+
+# Project Committee Member
+
+## About the Project Committee
+
+The PRAGMA Project Committee (PC) is a community body that helps evaluate, recommend, and oversee projects joining PRAGMA. The PC ensures alignment with PRAGMA’s mission to advance open source development within the Cardano ecosystem and beyond.
+
+Working closely with the Administrative Board, the committee reviews project proposals, provides guidance projects, and upholds PRAGMA’s values of 
+transparency, meritocracy, and collaboration.
+
+## Responsibilities
+Project Committee members are expected to:
+ - Review and evaluate project applications for technical quality, governance readiness, and community value.
+ - Engage with project teams to clarify goals, milestones, and alignment with PRAGMA’s open source principles.
+ - Provide recommendations to the Administrative Board on admitting or rejecting project applicants.
+ - Offer periodic feedback on active Maintainer Committees and help improve onboarding and reporting practices.
+ - Participate in discussions and votes at scheduled PC meetings (typically once per month).
+ - Promote collaboration and mentorship across the PRAGMA ecosystem.
+
+## Desired Qualities
+We’re looking for individuals who:
+ - Have experience in open source software, blockchain development, community management, or governance.
+ - Understand (or are eager to learn about) Cardano’s ecosystem.
+ - Demonstrate objectivity, discretion, and professionalism in evaluating proposals.
+ - Can commit a few hours per month for reviews and coordination.
+ - Value collaboration, diversity of thought, and consensus-building.
+
```

Comment:
We list out qualities we *want* to see, should we list out qualities we *don't* want to see or disqualifications?